src/middlewared/middlewared/migration/0009_nvram_attr_vms.py

```diff
@@ -0,0 +1,38 @@
+import os
+import shutil
+
+from middlewared.plugins.vm.utils import FINAL_NVRAM_FOLDER_PATH, get_vm_nvram_file_name
+
+
+DEFAULT_NVRAM_FOLDER_PATH = '/var/lib/libvirt/qemu/nvram'
+LIBVIRT_QEMU_UID = 64055
+LIBVIRT_QEMU_GID = 64055
+
+
+def migrate(middleware):
```

Comment:
Do we persist `/var/lib/libvirt/qemu/nvram` between upgrades somehow? I don't understand in what case will the files exist there when we first boot a TN release where this code will be included.

Reply:
@themylogin yes we preserve them https://github.com/truenas/scale-build/pull/576